tasks.rb

```diff
@@ -2959,6 +2961,20 @@ def apply_hpxml_modification(hpxml_file, hpxml)
     hpxml.windows[3].exterior_shading_factor_winter = 1.0
     hpxml.windows[3].interior_shading_factor_summer = 0.0
     hpxml.windows[3].interior_shading_factor_winter = 1.0
+  elsif ['base-enclosure-skylights-storms.xml'].include? hpxml_file
+    hpxml.skylights[0].ufactor = 0.6
+    hpxml.skylights[0].storm_window_type = 'clear'
+    hpxml.skylights[1].ufactor = 0.6
+    hpxml.skylights[1].storm_window_type = 'clear'
+  elsif ['base-enclosure-windows-storms.xml'].include? hpxml_file
+    hpxml.windows[0].ufactor = 0.6
+    hpxml.windows[0].storm_window_type = 'low-e'
+    hpxml.windows[1].ufactor = 0.6
+    hpxml.windows[1].storm_window_type = 'low-e'
+    hpxml.windows[2].ufactor = 0.6
+    hpxml.windows[2].storm_window_type = 'low-e'
+    hpxml.windows[3].ufactor = 0.6
+    hpxml.windows[3].storm_window_type = 'low-e'
```

Comment:
Unless we're asking for advanced/researchy/complex information, we should expose these inputs in the BuildResidentialHPXML measure and generate the sample files using that measure. (By using it to create the sample files, it also helps to test the measure.) In this way, users of the BuildResidentialHPXML measure (like ResStock) will have access to your new storm window model.

Let me or Joe know if you need help understanding how to make this change.